torchvision/prototype/models/_api.py

```diff
@@ -59,9 +60,14 @@ def verify(cls, obj: Any) -> Any:
 
     @classmethod
     def from_str(cls, value: str) -> "WeightsEnum":
-        for v in cls:
-            if v._name_ == value or (value == "default" and v.default):
-                return v
+        if value == "default":
+            default_value = getattr(cls, "default", None)
+            if default_value is not None:
+                return default_value
+        else:
+            for v in cls:
+                if v._name_ == value:
+                    return v
```

Comment:
I think we can simplify this with something like

```suggestion
        for v in cls.__members__:  # __members__ will include the "default" field which is an alias
            if field == value:
                return v
```

Reply:
I agree it can be simplified with:
```python
        for k, v in cls.__members__.items():
            if k == value:
                return v
```